include/ensmallen_bits/problems/fonseca_fleming_function.hpp

```diff
@@ -77,6 +78,12 @@ class FonsecaFlemingFunction
              -pow(static_cast<double>(coords[2]) - 1.0 / sqrt(3.0), 2.0)
         );
     }
+    arma::Col<size_t> Shuffle(size_t& populationSize)
+    {
+      return arma::shuffle(arma::linspace<arma::Col<size_t> >(0,
+            populationSize -1));
```

Comment:
```suggestion
          populationSize -1));
```

Should be tabbed twice (4 spaces) - https://github.com/mlpack/mlpack/wiki/DesignGuidelines#line-length-and-wrapping